### PR TITLE
Feature/feature 1041 personnel api model legacy

### PR DESCRIPTION
**Pull request by goeman**
Nothing stood out to be as problematic. I did add a field "ImportedDateTime" to track when the records were being inserted. Maybe we want a separate collection to track that? Or more fields such as a batchId?

### CODE REVIEW

File: models/personnel-import.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  PersonnelNote: {
+    type: String
+  },
+  DepartmentId: {
+    type: String
+  },
+  UnitID: {
+    type: String
+  },
+  ShiftStartTime: {
```
Comment (bamse16) [inline models/personnel-import.js]: ShiftStartDateTime
ShiftEndDateTime

File: models/personnel-import.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+var mongoose = require("mongoose");
+var Schema = mongoose.Schema;
+
+var modelSchema = new Schema({
+  _id: {
+    type: Schema.ObjectId,
+    auto: true
+  },
+  PersonnelID: {
```
Comment (bamse16) [inline models/personnel-import.js]: Mandatory

File: models/personnel-import.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    type: String
+  },
+  PersonnelName: {
```
Comment (bamse16) [inline models/personnel-import.js]: Mandatory

Comment (goeman) [reply models/personnel-import.js]: PersonnelID set to required: true

Comment (goeman) [reply models/personnel-import.js]: PersonnelName set to required: true

**Pull request merged**